user_manual/android_app.rst

```diff
@@ -239,6 +251,42 @@ Just ask them to copy and paste and send it to you.
 .. figure:: images/android-14.png
    :alt: Federated share creation.
    
+Working With Multiple Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Android application can perform some operations on multiple files simultaneously, such as refreshing and deleting. 
+To select multiple files, long select the first file that you want to work with; you will see a checkbox appear on the far right-hand side. 
+After that, check the checkbox next to all the other files that you want to perform the same operation on, and then perform the operation.
+
+.. figure:: images/android-select-multiple-files.png
+   :alt: Select multiple files.
+   
+Uploading Files Taken From the Camera
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Pictures and videos can be uploaded from your smartphone after choosing the
+folder where they are stored.
+To specify where they are located, in the "Settings" options, first enable one of "Instant picture uploads" or "Instant video uploads". 
+After that, a further option called "Camera folder" will become visible, as in the screenshot below. 
+
+.. figure:: images/specify-camera-folder.png
+   :alt: Specify camera folder.
+   
+Make Folders Available Offline 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Folders can be made available when no internet or mobile connectivity is available.
+Doing so caches a copy of the folder and its contents locally to the device, assuming that sufficient disc space is available.
```

Comment:
I would include here also: Doing so every change that appear in the folder included in the server, will be updated in the local folder without having to open it to refresh it.

Reply:
I will specify as well the posibility of keeping or removing the taken pic from the source folder

Reply:
@davigonz, that seems counter-intuitive. If a folder is available offline, how can changes made on the server automatically be made available in the local folder?

Reply:
@jesmrec, can you explain what you mean in more detail please?

Reply:
> that seems counter-intuitive. If a folder is available offline, how can changes made on the server automatically be made available in the local folder?

@settermjd what I meant is that folders are automatically updated even when you have not browsed to it yet, but that update is made when there's connection. Thus, when the user lose completely the connection, the folders contain synced data.

Reply:
@settermjd i mean, that the user can choose two options: remove or keep the pic/video from original folder after taking it. If he chooses "keep", the pic will remain in the camera folder and it will be uploaded to ownCloud. If he chooses "move" the pic will be deleted from camera folder after being uploaded to ownCloud.